libs/core/src/components/icon/icon.ts

```diff
@@ -1,127 +1,26 @@
-import { unsafeCSS } from 'lit'
-import { gdsCustomElement, html } from '../../scoping'
-import { GdsElement } from '../../gds-element'
+import { html, LitElement } from 'lit'
+import { property } from 'lit/decorators.js'
+import { unsafeHTML } from 'lit/directives/unsafe-html.js'
 
-import styles from './stem.styles.scss'
+export abstract class GdsIcon extends LitElement {
+  @property({ type: String }) width = 24
```

Comment:
Type should be `Number` here, right?
Or the value should be `"24"`

Reply:
Good catch changed the numbers to string ✌️